src/docs/arch-session.dox

```diff
@@ -53,24 +53,23 @@ rolling back any active transaction and closing the cursors that are still open.
 A session is always executed as a single thread, see @ref threads for more details.
 
 @section arch_session_error Error and Sub-level Errors Handling
-A session can track the return value of the prior API call through calling WT_SESSION::get_last_error. Alongside,
-sub-level error codes and messages are also accessible through calling the function providing contextual information 
-behind the call. The intended usage of the sub-level error code is to provide users more contextual information 
-on why an error has occurred, and allow users to programmatically make decisions based on sub-level error code.
+The WT_SESSION::get_last_error can be utilized to provide more verbose information about the last
+session API call. The information will include a sub-level error code and message providing
+contextual information. The intended usage of the sub-level error code is to allow users to
+programmatically make decisions on the application level.
 
-This performed through the WT_ERROR_INFO structure which tracks the previous session API call. The 
-WT_SESSION::get_last_error will return back any information recorded within the WT_ERROR_INFO structure. 
-An internal function set_last_error is utilised throughout the codebase to track any errors that may
-occur within WiredTiger. Developers should decide an appropiate time to utilise the WT_RET_SUB and 
-WT_RET_SUB macros when they need to provide more contextual information to the user on why an error has
-happened.
+Internal to WiredTiger the WT_ERROR_INFO structure lives within the session object and
+tracks any errors that occur during a session API call. The WT_SESSION::get_last_error will return
```

Comment:
We should be careful with "any errors", it does not track all of them.

Reply:
Is simply removing the word 'any' a sufficient change here?

Reply:
Removed now [5058d85](https://github.com/wiredtiger/wiredtiger/pull/11427/commits/5058d85dc0806548b7d763abfb402c0bc84f4be2)